src/__tests__/__reducers__/createArticle.test.js

```diff
@@ -1,83 +1,73 @@
 import reducer from "../../redux/reducers/createArticleReducer";
 import {
   NEW_ARTICLE,
-  INPUT_CHANGE,
   SUBMITTING_ARTICLE,
   FETCH_ARTICLE_TO_EDIT,
   ARTICLE_UPDATED,
-  ARTICLE_ERROR,
-  REMOVE_TAG,
-  NEW_TAG
+  ARTICLE_ERROR
 } from "../../redux/actionTypes";
 
 describe("article reducer", () => {
   test("should return the initial state on no action", () => {
     expect(reducer(undefined, {})).toEqual({
-      title: "",
-      description: "",
-      tagsList: [],
-      body: ""
+      article: {
+        title: "",
+        description: "",
+        tagsList: [],
+        body: ""
+      }
     });
   });
   test("should return the initial state on wrong action", () => {
     expect(
       reducer(undefined, { type: "ADD TITLE", payload: "hello world" })
```

Comment:
ADD TITLE or ADD_TITTLE

Reply:
@espoirMur I hope that you read the description of the test `should return the initial state when wrong action is dispatched`. Therefore, it is just a random action.